Update notifications.md (#8707)
DataDog/documentation: content/en/monitors/notifications.md

{{[dot.key.test].name}}
```

> #### Log facet for variables

Reviewer: ```suggestion
#### Log facet variables
```

> In Log monitor, you can group by facets. In that case, that facets can be used as a variable. 

Reviewer: ```suggestion
Log monitors can use facets as variables if the monitor is grouped by the facets.
```

> For example, if your facet is `@facet` and your monitor is grouped by that, use:

Reviewer: ```suggestion
For example, if your log monitor is grouped by the facet `@facet`, the variable is:
```

```text
{{@facet.name}}
```

> If your facet is with periods, let's say `@facet.with.dot` use:

Reviewer: ```suggestion
If your facet has periods, use brackets around the facet, for example:
```
